dotnet-interactive.Tests/HttpApiTests.cs

```diff
@@ -246,6 +248,72 @@ public async Task unimplemented_lsp_handlers_return_empty_string(Language langua
             responseJson.Should().BeEmpty();
         }
 
+        [Fact]
+        public async Task lsp_methods_run_deferred_commands()
+        {
+            // declare a variable in deferred code
+            var kernelBase = GetServer().Kernel as KernelBase;
+            kernelBase.DeferCommand(new SubmitCode("var one = 1;"));
+
+            // it's not defined
+            var response = await GetServer().HttpClient.GetAsync($"/variables/{Language.CSharp.LanguageName()}/one");
+            response.StatusCode.Should().Be(HttpStatusCode.NotFound);
```

Comment:
This sanity check is not needed.

Reply:
Why not?  I want to ensure the test is in a valid state and that we haven't accidentally executed `var one = 1;` at this time.

Reply:
That should be a different test, then. Multiple assertions per test tends to reduce the usefulness of the tests for diagnosing problems.